druid-shell/src/platform/windows/util.rs

```diff
@@ -103,6 +105,28 @@ impl FromWide for [u16] {
     }
 }
 
+/// Converts a `Rect` to a winapi `RECT`.
+#[inline]
+pub fn rect_to_recti(rect: Rect) -> RECT {
```

Comment:
`pub(crate)` here, probably; the main advantage is we'll get compiler warnings in the future if these are unused. (Probably not an issue in practice since this module shouldn't be public, but good general strategy I think)